src/app/pages/task-calendar/replication/replication-list/replication-list.component.ts

```diff
@@ -14,7 +14,7 @@ import helptext from '../../../../helptext/task-calendar/replication/replication
 
 @Component({
   selector: 'app-replication-list',
-  template: '<entity-table [title]="title" [conf]="this"></entity-table>',
+  template: '<entity-table [title]=\'title\' [conf]=\'this\'></entity-table>',
```

Comment:
This line looks weird. Maybe we could enable avoidEscape for quotes? https://eslint.org/docs/rules/quotes#avoidescape

Reply:
Agreed, also breaks message extraction. As for avoidEscape, it's already on, so not really sure what happened here.